examples/default/copies.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
+        {
+          "@id" : "copy",
+          "wasAssociatedWith" : {
+            "@type" : "prov:SoftwareAgent",
```

Comment:
This could be indexed by type

Reply:
I dont see how ...
my gues : 
```
 {
      "prov:SoftwareAgent" : [
        {
             "@id" : "653467354367537gjsfhdgfhjsf67345523bcs",
             "@type" : "prov:SoftwareAgent",
             "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
             "label" : "SPM" 
       }
       ],
      "prov:Activity" : [
        {
          "@id" : "653467354367537gjsfhdgfhjsf67345523efdghs",
          "label" : "copy",
          "wasAssociatedWith" : "SPM"
          "startedAtTime" : "2019-10-10T10:00:00",
          "endedAtTime" : "2019-10-10T10:00:00",
          "used" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz"
        }
      ],
      "prov:Entity" : [
        {
          "@id" : "sjhgdfhjsgd63q54673gbdsjdfsqjnfd",
          "prov:atLocation": "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz",
          "generatedAt": "2019-10-10T10:00:00"
        },
        {
          "@id" : "sjhgdfhfdsqldjkfnlqkjndflqsjfdnlqd",
          "prob:atLocation" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/PREPROCESSING/FUNCTIONAL/*",
          "derivedFrom" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz",
          "attributedTo" : "SPM",
          "wasGeneratedBy" : "copy"
        }
      ]
    }
```

Reply:
Yes! That's exactly it. The only tweak would be to change `"wasAssociatedWith" : "SPM"` in `"wasAssociatedWith" : "653467354367537gjsfhdgfhjsf67345523bcs"`. Does that make sense?

Reply:
that makes sense but then what are labels used for ?
I thought they were here as "aliases" for later referencing

Reply:
Unfortunately, labels can't be used as aliases because there is no guarantee that they are unique. One example of usage would be to display the label rather than the identifier) when creating a graphical representation of the graph.